data/seasons/d2-season-info.ts

```diff
@@ -10,11 +10,13 @@ export enum D2SeasonEnum {
   SHADOWKEEP,
   DAWN,
   WORTHY,
+  REDACTED_11, // TODO: Update on verification
+  REDACTED_12, // TODO: Update on verification
 
-  __LENGTH, // This always needs to be last
+  __LENGTH__, // This always needs to be last
 }
 
-export const D2CurrentSeason: number = D2SeasonEnum.__LENGTH - 1;
+export const D2CurrentSeason: number = D2SeasonEnum.__LENGTH__ - 1;
```

Comment:
If you're adding seasons 11 and 12, will this be off by one?

Reply:
It is an unused variable, hence the linked PR that has been merged